dosomething/main.tf

```diff
@@ -69,8 +69,8 @@ module "bertly" {
 
   environment   = "production"
   name          = "dosomething-bertly"
-  domain        = "new.dosome.click"
-  certificate   = "*.dosome.click"
+  domain        = "dosome.click"
+  certificate   = "dosome.click"
```

Comment:
Is the old Bertly still somewhere using `dosome.click`?

Reply:
Yep! I'm going to detach that mapping before I apply this. :link:

Reply:
Ah so that one is outside of terraform?

Reply:
Sounds good though - now I'm just curious!

Reply:
Yes! Originally, Bertly was managed with [CloudFormation](https://aws.amazon.com/cloudformation/), Amazon's configuration-as-code tool. It had some nice integrations in the AWS web GUI, but also only worked with AWS services which was a deal-breaker for a lot of our stuff.